lib/elixir/lib/calendar.ex

```diff
@@ -876,13 +1103,10 @@ defmodule NaiveDateTime do
       true
 
   """
-  @spec utc_now() :: t
-  def utc_now() do
-    {:ok, {year, month, day}, {hour, minute, second}, microsecond} =
-      Calendar.ISO.from_unix(:os.system_time, :native)
-    %NaiveDateTime{year: year, month: month, day: day,
-                   hour: hour, minute: minute, second: second,
-                   microsecond: microsecond}
+  @spec utc_now(Calendar.calendar) :: t
+  def utc_now(calendar \\ Calendar.ISO) do
+    DateTime.utc_now(calendar)
+    |> DateTime.to_naive
```

Comment:
Same as above. Let's make this mirror `Time.utc_now` by using tuples + convert. It will be the fastest for the most common use case.